app/models/manageiq/providers/ovirt/network_manager.rb

```diff
@@ -13,6 +13,9 @@ class ManageIQ::Providers::Ovirt::NetworkManager < ManageIQ::Providers::NetworkM
   include ManageIQ::Providers::Openstack::ManagerMixin
   include SupportsFeatureMixin
 
+  supports :create_network_router
+  supports :cloud_subnet_create
```

Comment:
NOTE It appears that the UI uses this feature to test for support of all of the network create features, there is no `create_cloud_network` feature.

IMO we should:
1. Add a `create_cloud_network` feature to core and all supporting providers
2. Update the UI to check for the right feature for the action being taken
3. Standardize on `create_cloud_subnet` since it is reversed compared with `create_network_router`

Reply:
can we use `CloudNetwork.supports :create` instead of putting that here?

Reply:
Yeah definitely, you mean like https://github.com/ManageIQ/manageiq/blob/master/app/models/ext_management_system.rb#L303 (with `:child_model`)

Reply:
@kbrock do you want to do that in this PR?  I was thinking we would add the functionality the way it is currently implemented in this provider, then follow-up with refactoring